server/router.go

```diff
@@ -8,4 +8,6 @@ func setupRoutes(router *gin.Engine) {
 
 	v1.GET("/", hello)
 	v1.POST("/", dummyPost)
+	v1.GET("/nearbytaxi", getNearByTaxi)
```

Comment:
Can we make it nearByTaxis ? as we are returning multiple taxis

Reply:
Done